airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml

```diff
@@ -2041,7 +2041,7 @@
 - name: Stripe
   sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   dockerRepository: airbyte/source-stripe
-  dockerImageTag: 3.4.3
+  dockerImageTag: 3.5.0
   documentationUrl: https://docs.airbyte.com/integrations/sources/stripe
   icon: stripe.svg
   sourceType: api
```

Comment:
Can you delete this file? We're no longer using this file on master and instead are relying on metadata.yaml moving forward.